webapp-django/crashstats/cron/__init__.py

```diff
@@ -48,6 +48,12 @@
         "time": "04:00",
         "backfill": True,
     },
+    {
+        # Check past missing crashes and update status daily at 3:00am
+        "cmd": "updatemissing",
+        "frequency": "1d",
+        "time": "03:00",
```

Comment:
Running it at 3:00am is sort of an arbitrary time. We could run it afterwards. I'm not sure it matters a whole lot.